packages/salesforcedx-vscode-core/src/commands/startApexDebugLogging.ts

```diff
@@ -118,7 +118,8 @@ export class StartApexDebugLoggingExecutor extends SfdxCommandletExecutor<{}> {
 }
 
 export async function getUserId(projectPath: string): Promise<string> {
-  const defaultUsernameOrAlias = await workspaceContextUtils.getDefaultUsernameOrAlias();
+  const defaultUsernameOrAlias =
```

Comment:
Change defaultUsername to targetOrg everywhere in this file to match the CLI flag?